packages/contracts/test/test-utils/oz-constants.ts

```diff
@@ -2,3 +2,6 @@ export const OZ_ERRORS = {
   ALREADY_INITIALIZED: 'Initializable: contract is already initialized',
   ERC20_INSUFFICIENT_ALLOWANCE: 'ERC20: insufficient allowance',
 };
+
+export const IMPLEMENTATION_SLOT =
```

Comment:
I wouldn't rename the file to something including OZ because the storage slot is defined in the EIP and not by OZ
https://eips.ethereum.org/EIPS/eip-1967

Reply:
Changed in 02711cc0834252722eace1e18e398c8bf75c691d